core/base/src/TSystem.cxx

```diff
@@ -1852,21 +1852,11 @@ int TSystem::Load(const char *module, const char *entry, Bool_t system)
       if (!system) {
          // Mark the library in $ROOTSYS/lib as system.
          const char *dirname = DirName(path);
-#ifdef ROOTLIBDIR
-         TString rootlibdir = ROOTLIBDIR;
-#else
-         TString rootlibdir = "lib";
-         PrependPathName(gRootDir, rootlibdir);
-#endif
+         TString rootlibdir = TROOT::GetLibDir();
          system = R__MatchFilename(rootlibdir,dirname);
```

Comment:
Can we just call `system = R__MatchFilename(TROOT::GetLibDir(), dirname);` or do we need the temporary?